README.md

```diff
@@ -0,0 +1,568 @@
+```
+
+By default, PhD uses the source code highlighter that is built into PHP
+itself which is only able to highlight PHP code.
+
+If your documentation contains other types of source code or markup,
+like XML, HTML, Javascript or any other language, you should try the
+[GeSHi](http://qbnz.com/highlighter/) highlighter that is shipped with
+PhD:
+
+1.  Install GeSHi from the MediaWiki PEAR channel:
+
+```shell
+$ pear channel-discover mediawiki.googlecode.com/svn
+$ pear install mediawiki/geshi
+```
+
+2.  Use the GeSHi syntax highlighting class when rendering your
+    documentation:
+
+```shell
+$ phd -g 'phpdotnet\phd\Highlighter_GeSHi' -d phd-guide.xml
+```
+
+If you have GeSHi version 1.1.x installed, you should use the
+`phpdotnet\phd\Highlighter_GeSHi11x` highlighter, which is adapted to
+GeSHi's new API.
+
+Apart from using the highlighter shipped with PhD, you can [build your
+own highlighters](#writing-a-custom-syntax-highlighter).
+
+# DocBook extensions
+
+PhD has been tailored for PHP and PEAR manuals. To make writing
+documentation as easy as possible, some custom tags have been added to the
+DTD.
+
+All extensions live in their own XML namespace "`phd:`" which resolves
+to <http://www.php.net/ns/phd>. When using one of the attributes or tags
+described here, remember to set the namespace:
+
+```xml
+xmlns:phd="http://www.php.net/ns/phd"
+```
+
+## General DocBook extensions
+
+The extensions listed here are available in all PhD themes and formats.
+
+### Manual chunking with "phd:chunk" (Attribute)
+
+PhD automatically chooses which sections, chapters or other tags get
+their own file (chunk) when using a chunked theme. Sometimes the result
+of this automatism is not optimal and you want to fine-tune it. The
+attribute "phd:chunk" is offered as a solution by PhD.
+
+#### Allowed values
+
+The value of `phd:chunk` can only be `true` or `false` which force the element
+to be chunked or not.
+
+#### Allowed in
+
+`phd:chunk` may be used in every tag that accepts
+[db.common.attributes](https://tdg.docbook.org/tdg/5.0/ref-elements.html#common.attributes).
+
+#### Example
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<preface xmlns="http://docbook.org/ns/docbook"
+    xmlns:phd="http://www.php.net/ns/phd"
+    xml:id="preface"
+    phd:chunk="false"
+>
+    <info>
+    <title>Preface</title>
+    ..
+    </info>
+    ..
+</preface>
+```
+
+
+### Generating Table Of Contents: \<phd:toc\> (Tag)
+
+This tag can be used to manually insert a Table Of Contents (TOC)
+that creates a list of links to children elements of a tag.
+
+phd:toc-depth
+
+#### Allowed in
+
+`<phd:toc>` can be used everywhere `<para>` elements are allowed.
+
+#### Children
+
+You can add a title with `<title>`.
+
+#### Attributes
+
+| Attribute name | Description | Default value |
+|----|----|----|
+| phd:element | ID of the element whose children shall be linked | *none* |
+| phd:toc-depth | Depth of the TOC/Number of levels | `1` |
+
+Attributes for \<phd:toc\>
```

Comment:
Does this even work? Also conflicts with standard docbook (and really should be a PI if supported)

Reply:
This is not used in `doc-en`. I'll remove it from the README.